test/test_mps.py

```diff
@@ -1,5 +1,5 @@
 # Owner(s): ["module: mps"]
-
+# ruff: noqa: F841
```

Comment:
cc @kulinseth we should enable this one to make sure there is no hidden bug here.